README.md

```diff
@@ -1,9 +1,234 @@
-The Write-API allows developer to automatically create GitHub and Quay repositories for any tool encapsulated in Docker and described with a CWL descriptor on behalf of the user.
-It implements the proposed GA4GH [Write API](https://github.com/ga4gh/tool-registry-schemas/blob/feature/write_api_presentation/src/main/resources/swagger/ga4gh-tool-discovery.yaml)
+![Build Status](https://travis-ci.org/dockstore/write_api_service.svg?branch=develop)
+
+# Write API Service and Client
+
+This is a service aimed at two tasks
+1. Providing a concrete reference implementation of a proposed GA4GH [Write API](https://github.com/ga4gh/tool-registry-schemas/blob/feature/write_api_presentation/src/main/resources/swagger/ga4gh-tool-discovery.yaml).
+2. Providing a utility for developers to convert plain CWL/WDL files and Dockerfiles into [GitHub](https://github.com) repos storing those plain CWL/WDL files and [Quay.io](https://quay.io) repos storing Docker images built from those Dockerfiles. This can be used by those converting tools described in other formats into "Dockstore-friendly" tools that can be quickly registered and published in Dockstore by using the Write API Client's publish command or programmatically via the Dockstore API.  It is an alternative to using a GUI to register tools on Dockstore.
+
+## End Users
+This is intended to be used by:
+- Tool Migrators
+
+  Developers that have access to a large number of tools in some different format and wants to migrate them all programmatically to Dockstore with minimal effort.
+- Tool Developers
```

Comment:
Realistically, I think this would be much more applicable for teams converting many tools than someone creating just one tool.